services/housetables/src/main/resources/data.sql

```diff
@@ -0,0 +1,4 @@
+-- Initial value for feature toggle tables
+-- When enabling/disabling some feature, please ensure they are checked-in and reviewed through this file
+
+INSERT IGNORE INTO table_toggle_rule (feature, database_pattern, table_pattern, id, creation_time_ms) VALUES ('demo', 'demodb', 'demotable', DEFAULT, DEFAULT);
```

Comment:
what happens if we just do "INSERT INTO", conflict?

Reply:
yes. Because of the unique constraints declared in DDL, insertion will fail.